src/main/java/models/Player.java

```diff
@@ -0,0 +1,18 @@
+package models;
+
+public class Player {
+
+    private String name;
+    private int score;
+
+    public Player(String name, int score){
+        this.name = name;
+        this.score = score;
+    }
+
+    public int getScore() {
```

Comment:
omit the space: `getScore(){` for consistency :)

Reply:
+1